crates/polars-pipe/src/pipeline/convert.rs

```diff
@@ -109,7 +109,7 @@ where
                 FileScan::Csv { options, .. } => {
                     let src = sources::CsvSource::new(
                         sources,
-                        file_info.schema,
```

Comment:
Accidentally passed the full schema instead of the reader schema, causing the column-shifting issue